src/content/docs/en/upgrade-astro.mdx

```diff
@@ -180,3 +180,24 @@ A major release may also include some non-breaking changes and improvements that
   Once these features are out of their experimental period, they will follow the normal semantic versioning contract.
 
 - **Improvements to the documentation** (e.g. reference and error messages). They are built from source for the `docs` repository. This allows Astro to quickly update docs fixes and improvements in the cases where documentation source content is stored in the main `astro` repository.
+
+### Node.js support and upgrade policies
+
+#### Support
+
+- Astro supports the **latest _Maintenance_ LTS** versions of Node.js.
```

Comment:
```suggestion
- Astro supports the [**latest _Maintenance_ LTS** versions of Node.js]([nodejs.org/en/about/previous-releases#release-schedule](https://nodejs.org/en/about/previous-releases#release-schedule).
```
I think the link here makes sense!

Reply:
There is an extra portion in the middle.

```suggestion
- Astro supports the [**latest _Maintenance_ LTS** versions of Node.js](https://nodejs.org/en/about/previous-releases#release-schedule).
```